lib/puppetserver/ca/cli.rb

```diff
@@ -22,10 +26,13 @@ class Cli
 
       ACTION_OPTIONS = "\nAction Options:\n" +
         VALID_ACTIONS.map do |action, cls|
-          "  #{action}:\n" +
-          cls.parser.summarize.
+          summary = cls.parser.summarize.
             select{|line| line =~ /^\s*--/ }.
             reject{|line| line =~ /--help|--version/ }.join('')
+
```

Comment:
@justinstoller you did something similar, which is probably better.  I just left this in for now.